en/middleware/xrce_dds.md

```diff
@@ -241,23 +244,90 @@ The topics are release specific (support is compiled into [microdds-client](../m
+The definition are stored in the ROS 2 interface package [PX4/px4_msgs](https://github.com/PX4/px4_msgs) and they are automatically synchronized by CI on the `main` and release branches.
+Therefore,
 
-- If you're using a main or release version of PX4 you can get these by cloning [PX4/px4_msgs](https://github.com/PX4/px4_msgs) into your workspace (branches are provided for each release).
-- If you're creating your own new topic you will need to update [dds_topics.yaml](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/microdds_client/dds_topics.yaml) and also copy the message definitions out of PX4 source tree into any ROS 2 workspace that needs them.
-  The messages must be in a folder named `px4_msgs/msg/`.
+- If you're using a main or release version of PX4 you can get the message definitions by cloning the interface package [PX4/px4_msgs](https://github.com/PX4/px4_msgs) into your workspace.
+- If you're creating your own new topics e/o uORB messages you will need to update [dds_topics.yaml](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/microdds_client/dds_topics.yaml), clone the the interface package and then manually synchronize it by copying the new/modified message definitions from [PX4-Autopilot/msg](https://github.com/PX4/PX4-Autopilot/tree/main/msg) to its `msg` folders. Assuming that PX4-Autopilot is in your home directory `~` while `px4_msgs` is in `~/px4_ros_com/src/`, then the command can be
+  ```sh
+  rm ~/px4_ros_com/src/px4_msgs/msg/*.msg
+  cp ~/PX4-Autopilot/mgs/*.msg ~/px4_ros_com/src/px4_msgs/msg/
+  ```
   
   :::note
-  Technically the messages must be in a folder defined by the `type` prefix in the yaml file:
-  As you can see below, this is `px4_msgs::msg::`:
-  
-  ```yaml
-  - topic: /fmu/out/vehicle_odometry
-    type: px4_msgs::msg::VehicleOdometry
+  Technically, [dds_topics.yaml](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/microdds_client/dds_topics.yaml) completely defines the pairing between PX4 message and topics and the ROS 2 ones.
+  It is structured as follows:
+  ```
+  publications:
+
+    - topic: /fmu/out/collision_constraints
+      type: px4_msgs::msg::CollisionConstraints
+
+    ...  
+
+    - topic: /fmu/out/vehicle_odometry
+      type: px4_msgs::msg::VehicleOdometry
+
+    - topic: /fmu/out/vehicle_status
+      type: px4_msgs::msg::VehicleStatus
+
+    - topic: /fmu/out/vehicle_trajectory_waypoint_desired
+      type: px4_msgs::msg::VehicleTrajectoryWaypoint
+
+  subscriptions:
+
+    - topic: /fmu/in/offboard_control_mode
+      type: px4_msgs::msg::OffboardControlMode
+
+    ...
+
+    - topic: /fmu/in/vehicle_trajectory_waypoint
+      type: px4_msgs::msg::VehicleTrajectoryWaypoint
   ```
+  
+  Each (`topic`,`type`) pairs defines
+
+  1. A new subscription or publication depending on the list it is added.
+  2. The topic _base name_, which MUST coincide with the desired uORB topic name that you want to publish/subscribe.
```

Comment:
FYI There is nothing wrong with highlighting MUST, but I prefer to use bold/italic, and have done so across the docs. There are exceptions.

```suggestion
  2. The topic _base name_, which **must** coincide with the desired uORB topic name that you want to publish/subscribe.
```